### PR TITLE
Add jotcode

### PR DESCRIPTION
**Pull request by aclarembeau**
Add Jotcode as a PAAS provider for simple scripts and APIs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -922,6 +922,7 @@ Table of Contents
   * [ddos-guard.net](https://ddos-guard.net/store/web) — Free CDN, DDoS protection and SSL certificate
   * [developers.google.com](https://developers.google.com/speed/libraries/) — The Google Hosted Libraries is a content distribution network for the most popular, Open Source JavaScript libraries
   * [GraphCDN](https://graphcdn.io/) - Scale, inspect & protect your GraphQL API. 5m requests per month for free.
+  * [Jotcode](https://jotcode.io) - A serverless platform for hosting simple scripts, API and servers
```
Comment (ripienaar) [inline README.md:925]: Please include someting like `free plan allows running 100 scripts per month`

Comment (aclarembeau) [reply README.md]: Sure. I just did it :) Thanks

**Comment by ripienaar** [comment]
thanka

**Comment by imba-tjd** [comment]
Why putting it under `CDN and Protection`?

**Pull request merged**